lib/controllers/root-controller.js

```diff
@@ -488,8 +497,30 @@ export default class RootController extends React.Component {
 
   async viewChangesForCurrentFile(stagingStatus) {
     const editor = this.props.workspace.getActiveTextEditor();
-    const absFilePath = editor.getPath();
+    const absFilePath = await realPath(editor.getPath());
     const repoPath = this.props.repository.getWorkingDirectoryPath();
+    if (repoPath === null) {
+      const [projectPath] = this.props.project.relativizePath(editor.getPath());
+      const notification = this.props.notificationManager.addInfo(
+        "Hmm, we don't have anything to compare this to.",
```

Comment:
@jasonrudolph Thanks for the reminder, I'll see if I can find some more infos around this topic.